src/ResourceManager/EventGrid/Commands.EventGrid/Utilities/EventGridUtils.cs

```diff
@@ -47,7 +49,7 @@ public static string GetScope(string subscriptionId, string resourceGroupName, s
                 throw new ArgumentNullException(nameof(subscriptionId));
             }
 
-            string scope;
+            string scope = null;
```

Comment:
`string` is null at declaration (meaning `= null` does nothing). Just a comment; no change required.